### PR TITLE
feat(project.js): Template task option added in repair order.

### LINKED ISSUE #339: feat(Repair Order): Template Task option in Repair Order
Add create template task and task button in repair order. 
When the user clicks on this button, a template task will be created and will shown in the vehicle control panel.

### PR DESCRIPTION
**Pull request by ebbad18**
closes: https://github.com/ParaLogicTech/erpnext/issues/339

### CODE REVIEW

**Review by ebbad18 — COMMENTED** [review]
.

File: erpnext/controllers/item_variant.py
Comment (ebbad18) [inline erpnext/controllers/item_variant.py:1]: Dont pull this file, discard it.

Comment (SaiFi0102) [reply erpnext/controllers/item_variant.py:1]: Revert the file if you don't want this pulled, don't just ask for it, do it

File: erpnext/public/js/utils/projects.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	task_doc.subject = subject;
+	task_doc.expected_time = flt(standard_time);
+	task_doc.project_template = project_template;
+
+	task_doc.save(null, () => {
+		frappe.msgprint(_("{0} created").format(frappe.get_link(task_doc)), { indicator: "green" });
+	});
+};
+
+erpnext.projects.create_task
```
Comment (SaiFi0102) [inline erpnext/public/js/utils/projects.js]: Remove this line

**Review by SaiFi0102 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**